### PR TITLE
html-forms exercise - Heerak Moon

### PR DESCRIPTION
**Pull request by hmoon2**
https://user-images.githubusercontent.com/113075380/206015351-3292e533-e2aa-4926-a3c7-502752c21eeb.mp4

### CODE REVIEW

File: html-forms/index.html [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+<html>
+  <head>
+    <title>HTML Forms</title>
+  </head>
+  <body>
+    <h1>Coding Forms!</h1>
+    <form action="index.html" method="post">
+      <label for="email-address">Email</label>
+      <div>
+        <input type="text" name="email">
```
Comment (codingcodymiller) [inline html-forms/index.html:11]: This needs an `id` that matches the `for` on its `label` to be properly configured.

File: html-forms/index.html [hunk 2]
```diff
@@ -0,0 +1,45 @@
...
+      </div>
+      <label for="coding">Why is coding cool?</label>
+      <div>
+        <select name="options">
```
Comment (codingcodymiller) [inline html-forms/index.html:15]: This needs an `id` that matches the `for` on its `label` to be properly configured.

File: html-forms/index.html [hunk 3]
```diff
@@ -0,0 +1,45 @@
...
+      </label>
+      <p>Would you like to sign up for more information?</p>
+      <label>
+        <input type="checkbox" name="yes" value="Sign-up">Sign me up!
+      </label>
+      <div>
+        <label for="comments">Comments</label>
+      </div>
+      <div>
+        <textarea rows="10" cols="35"></textarea>
```
Comment (codingcodymiller) [inline html-forms/index.html:40]: This needs an `id` that matches the `for` on its `label` to be properly configured.

**Review by codingcodymiller — CHANGES_REQUESTED** [review]

**Comment by hmoon2** [comment]
html-forms-fix

**Review by codingcodymiller — APPROVED** [review]

**Pull request merged**